src/cli/utils/sharepoint.go

```diff
@@ -148,7 +146,7 @@ func IncludeSharePointRestoreDataSelectors(ctx context.Context, opts SharePointO
 
 	lfp, lfn := len(opts.FolderPath), len(opts.FileName)
 	ls, lwu := len(opts.SiteID), len(opts.WebURL)
-	slp, sli := len(opts.ListFolder), len(opts.ListItem)
+	sl := len(opts.Lists)
```

Comment:
Semi-related to this PR but could you rename these variables to follow the same scheme as in groups (around line 182 above)? It'll make it less bug-prone.